src/test/java/gov/hvtesting/Features/OperatorJourney.feature

```diff
@@ -1,23 +1,15 @@
 @OperatorJourney
 Feature: Operator Journey
 
-  @UI @TC12
+  @UI
   Scenario: Operator can search for nearest ATF in UI and go back to search page
     Given I go to Nearest ATF Search page
     When I fill in my postcode with "AB101AA"
     Then I am on Nearest ATF Results page for "AB10 1AA"
     When I click back button
     Then I am on Nearest ATF Search page
-
-
-  @UI @TC13
```

Comment:
We will need to revisit this test and either refactor or remove any unused steps if the test is no longer needed